src/Application/Interfaces/IMediator.cs

```diff
@@ -0,0 +1,6 @@
+namespace Application.Interfaces;
+
+public interface IMediator
```

Comment:
you should add a reference to the Mediator package to the Application layer, and use Mediator.IMediator

Reply:
should i use abstraction

Reply:
no, use the `Mediator.IMediator` interface